### DISCUSSION TITLE
Does this game have any future?

### DISCUSSION THREAD

**Discussion by Waulfree** [redeclipse/base#1129] [Help and Support] [answered]
I like red eclipse, it looks pretty nice from many aspects. But it's unpopular afaik. Could that perspective be changed in future somehow?
That seem's like an issue for me, because i can't enjoy the game that have no players online.

**Comment by IceflowRE** [comment]
It was already discussed here: https://redeclipse.net/oldforum/viewtopic.php?f=4&t=848, maybe you find something interesting or continue the topic here.

**Comment by qreeves** [comment, marked as answer]
With the dwindling supply of open source first person shooters under active development, we're actually seeing an increase in activity elsewhere in our community. At the moment, part of it is just playing at the right times (usually the weekends).

That being said, we're [working on v2.0 at the moment](http://www.redeclipse.net/docs/Information-for-v2-0), which will include a lot of changes both to the rendering system and the overall game experience. We're using this opportunity to see what works, what doesn't, and what's needed in order to improve how the game plays for everyone.

Once we get to that point, we're hoping to do a big marketing push and publish the game on Steam in order to access a wider audience. You can help with this by participating on the discussions when we propose new changes, helping us get our new website/documentation up to scratch, and porting old maps to make use of the new renderer.

**Comment by nok-ko** [comment]
I'm interested in contributing to the documentation efforts. Is there a list of all the pages from the old wiki that need moving/re-formatting to fit into the new website?

**Comment by IceflowRE** [comment]
Theres no list, but you can take a look into [docs](https://github.com/red-eclipse/docs) and the copy of the [wiki](https://github.com/red-eclipse/oldwiki).
But if you "copy" them please check that the information is not outdated. And check this https://github.com/red-eclipse/world/issues/9.
